nori.proto

```diff
@@ -1,8 +1,8 @@
 syntax = "proto3";
 
-import "plugin_messages/messages.proto";
-import "config_messages/config_messages.proto";
-import "common_messages/common_messages.proto";
+import "plugin/plugin.proto";
+import "config/config.proto";
+import "common/common.proto";
 
 
 
```

Comment:
Change import and package order. See style guide https://developers.google.com/protocol-buffers/docs/style